modules/account-lib/test/unit/coin/algo/utils.ts

```diff
@@ -21,4 +21,16 @@ describe('utils', () => {
     const objEncodedByAlgo = algosdk.encodeObj(note);
     should.deepEqual(objEncoded, objEncodedByAlgo);
   });
+
+  it('should return a mnemonic from a secret key ', () => {
```

Comment:
Add test for invalid cases adding an invalid secret key

Reply:
Same for invalid seed or keyPair, if its possible.